src/test/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSinkTest.java

```diff
@@ -466,6 +467,7 @@ private Context createContext(boolean useDeprecatedParams, String fileFormat) {
             context.put("hdfs_username", hdfsUsername);
         } // if else
         
+        context.put("csv_separator",csvSeparator);
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/d89500c46c3c725a627c2b47e0fe6214cef0d6ad